src/modules/layout/components/Header.js

```diff
@@ -171,9 +168,10 @@ class Header extends React.PureComponent<PropsType> {
           {cityModel && <HeaderText theme={theme}>{this.cityDisplayName(cityModel)}</HeaderText>}
         </HorizontalLeft>
         <MaterialHeaderButtons>
-          {this.renderItem('Search', 'search', 'always', !this.isPeeking() ? this.goToSearch : undefined)}
-          {this.renderItem('Change Language', 'language', 'always',
-            !this.isPeeking() && goToLanguageChange ? goToLanguageChange : undefined)}
+          {!peeking && categoriesAvailable &&
+            this.renderItem('Search', 'search', 'always', this.goToSearch)}
+          {!peeking && goToLanguageChange &&
+          this.renderItem('Change Language', 'language', 'always', goToLanguageChange)}
           {this.renderItem(t('share'), undefined, 'never', sharePath ? this.onShare : undefined)}
```

Comment:
you might just show this only if sharePath is defined

Reply:
https://issues.integreat-app.de/browse/NATIVE-417